docs/modules/release-notes/pages/5-5-1.adoc

```diff
@@ -0,0 +1,14 @@
+= 5.5.1 Release Notes
+:description: These release notes list any new features, enhancements, fixes, and breaking changes implemented between version 5.5.0 and version 5.5.1 of Hazelcast Platform.
+
+{description}
+
+== Known Issues
+* There is currently a known issue that has potential for causing a memory leak in Vector collections in some scenarios. For more info, including a workaround, see xref:data-structures:vector-collections.adoc#known-issue[Vector Collection Known Issue].
+
+== Fixes
+* Fix partition table snapshot selection (SUP-538)
```

Comment:
My suggestion: 
Partitions are not distributed correctly after all Members in a partition group are restarted and persistence is enabled